docker/ogl_obr.dockerfile

```diff
@@ -0,0 +1,58 @@
+SHELL ["/bin/bash", "-c"]    
+# We need to set this enviroment variable    
+# since different OpenFOAM versions have    
+# a different name for this case    
+ENV CYCLIC_CASE=periodicPlaneChannel    
+# Set the PATH variable for local pip installs    
+ENV PATH="${PATH}:${HOME}/.local/bin"    
+                                                                                                                                                                    
+# Set the default solver execution template                                                                                                                         
+# we set this to explicitely allow oversubscribe                                                                                                                    
+ENV OBR_RUN_CMD="mpirun --oversubscribe --bind-to core --map-by core -np {np} {solver} -parallel -case {path}/case >  {path}/case/{solver}_{timestamp}.log 2>&1"    
+                                                                                                                                                                    
+RUN sudo apt update && sudo apt install -y pip ninja-build    
+                                                              
+# update python environment                    
+RUN python3.10 -m pip install pip --upgrade            
+                                                       
+# Clone dependencies                                   
+RUN cd $HOME && \                                      
+    git clone https://github.com/hpsim/OGL.git && \    
+    git clone https://github.com/hpsim/OBR.git && \    
+    pip install ./OBR                                  
+                                                           
+                                                           
+# BUILD and install OGL                                             
+RUN cd $HOME/OGL && \                                               
+  source /usr/lib/openfoam/openfoam2206/etc/bashrc && \             
+  cmake --preset ninja-cpuonly-release && \                         
+  cmake --build --preset ninja-cpuonly-release  --target install    
+                                                                    
+# Create test cases                                               
+# this creates a workspace folder with all the required           
+# cases and variations as defined in the integration.yaml file    
+RUN mkdir -p $HOME/OGL_integration_tests &&  cd $_ && \           
+    source /usr/lib/openfoam/openfoam2206/etc/bashrc && \     
+    obr init --config $HOME/OGL/test/integration.yaml && \    
+    obr run -o generate                                       
+                           
+    
+# Execute all unit test cases                                                                                                                              
+# this might take a while since we set the debug flag    
+RUN obr run -o runParallelSolver    
```

Comment:
I get `ERROR: failed to solve: process "/bin/bash -c obr run -o runParallelSolver" did not complete successfully: exit code: 1` here; forgot to cd somewhere?

Reply:
`LookupError: Unable to find project at path '/home/openfoam'.`

Reply:
Yes, that is true. I need to finish that up. I was unsure whether to include that as a run step or let the user/reviewer do it manually. But I think i'll do both.